src/catchError.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable max-len */
+const massageError = [
+  { message: 'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".' },
+  { message: '"toCase" query param is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".' },
+  { message: 'This case is not supported. Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.' },
+];
+
+const cases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+export const catchError = (text, toCase) => {
+  const errors = [];
+
+  if (!text || text === '/') {
+    errors.push(massageError[0]);
```

Comment:
not a good idea to take elements by indexes, let us make `massageError` object, not array